js-miniapp-bridge/src/bridge.js

```diff
@@ -114,6 +114,24 @@ MiniAppBridge.requestPermission = function(permissionType) {
     );
   });
 };
+
+/**
+ Below code will override the navigator.geolocation.getCurrentPosition method for only iOS
+ */
+if (isPlatform.iOS()) {
+    navigator.geolocation.getCurrentPosition = function(success, error, options) {
+        return MiniAppBridge.exec(
+            "getCurrentPosition",
+            {locationOptions: options},
+            function(value) {
+                var parsedData = JSON.parse(value)
+                success(parsedData)
```

Comment:
Maybe the `JSON.parse` should be in a `try/catch`? I know that the iOS side is directly controlling what's returned here, but seems like a good idea in case of some edge case where one of the values can't be parsed correctly:

```suggestion
                try {
                    var parsedData = JSON.parse(value)
                    success(parsedData)
                } catch(error) {
                    error({
                        code: GeolocationPositionError.POSITION_UNAVAILABLE,
                        message: "Failed to parse location object from MiniAppBridge: " + error
                    })
                }
```

What do you think?